iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuizHints/Views/StepQuizHintCardView.swift

```diff
@@ -0,0 +1,134 @@
+import shared
+import SwiftUI
+
+extension StepQuizHintCardView {
+    struct Appearance {
+        let authorAvatarWidthHeight: CGFloat = 20
+        let reactionImageWidthHeight: CGFloat = 16
+    }
+}
+
+struct StepQuizHintCardView: View {
+    private(set) var appearance = Appearance()
+
+    let authorAvatarSource: String?
+
+    let authorName: String
+
+    let hintText: String
+
+    let hintHasReaction: Bool
+
+    let onHintReactionButtonTap: (ReactionType) -> Void
+
+    let onHintReportButtonTap: () -> Void
+
+    let onNextHintButtonTap: (() -> Void)?
+
+    @State private var isPresentingReportAlert = false
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: LayoutInsets.defaultInset) {
+            HStack {
+                LazyAvatarView(authorAvatarSource)
+                    .frame(widthHeight: appearance.authorAvatarWidthHeight)
+
+                Text(authorName)
+                    .font(.caption)
+                    .foregroundColor(Color(ColorPalette.black900Alpha38))
```

Comment:
В дизайне используется цвет `colorOnSurface_0.38` и у нас есть цвет для этого `Color.disabledText`.
Можно заменить на:
```suggestion
                    .foregroundColor(.disabledText)
```

И везде такие цвета пробегись замени если что.